tests/lit-tests/3141.ispc

```diff
@@ -0,0 +1,20 @@
+// RUN: not %{ispc} --nowrap --target=host --enable-llvm-intrinsics --emit-llvm-text -o - %s 2>&1 | FileCheck %s
+
+// REQUIRES: LLVM_17_0+
+
+#if LLVM_VERSION_MAJOR >= 19
```

Comment:
Do we really define `LLVM_VERSION_MAJOR` for tests? I believe we don't.

Reply:
We expose LLVM versions (PR #3120) for users, so this macro is defined.

Reply:
I think we can proceed with this PR and then do renaming in a separate one.